how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -0,0 +1,126 @@
+
+# Kata Containers with ACRN
+
+- [Introduction](#introduction)
+- [Pre-requisites](#pre-requisites)
+- [Install and Configure Docker](#install-and-configure-docker)
+- [Configure Kata Containers with ACRN](#configure-kata-containers-with-acrn)
+- [ACRN Reference](#acrn-reference)
+
+## Introduction
+
+ACRN is a flexible, lightweight reference hypervisor, built with real-time and safety-criticality in mind and optimized to streamline embedded development through an open-source platform.
+
+There’re two key components of ACRN: the hypervisor itself and the ACRN device model. The __ACRN Hypervisor__ is a Type 1 reference hypervisor stack, running directly on bare-metal, and is suitable for a variety of IoT and embedded device solutions.  ACRN hypervisor addresses the gap that currently exists between datacenter hypervisors and hard partitioning hypervisors. The ACRN hypervisor architecture partitions the system into different functional domains, with carefully selected guest OS sharing optimizations for IoT and embedded devices. The __ACRN Device Model__ is a reference framework implementation for virtual device emulation that provides rich I/O virtualization support currently planned for audio, video, graphics, and USB. More mediator features are expected as the community grows.
+
+## Pre-requisites
+
+This document requires ACRN hypervisor and Kata containers installed on your system. The following provide links to installation of ACRN hypervisor and Kata containers:
+
+- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
+- ACRN [software](https://projectacrn.github.io/latest/getting-started/apl-nuc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- Kata containers installation: Automated installation doesn't seem to be supported for clearlinux, so please use [manual installation](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md) steps.
+
+> **Note:** Create rootfs image and not initrd image.
+
+In order to run Kata + ACRN, your container stack must provide block-based storage ('graph driver'), such as device-mapper.
+
+> **Note:** Currently only 1 VM can be launched from kata-containers using ACRN hypervisor (SDC scenario) due to [this issue](https://github.com/kata-containers/runtime/issues/1785). Once the issue is addressed from kata, we can test with other post launched VM. Other scenarios (such as Industry, hybrid, and logical partitioning) are not supported yet.
+
+## Install and Configure Docker
+
+Docker 18.06 is required for running Kata with ACRN. For Kata+ACRN, a block-based driver like device-mapper is required. The latest release of Docker, 18.09, does not support device-mapper and is not compatible.
+
+To configure Docker for device-mapper and Kata, first, stop Docker daemon if it is already running using,
+
+```bash
```

Comment:
I'd make this a numbered list:
*Follow these instructions to configure Docker to use the device-mapper backend and the Kata runtime*
*1. stop the Docker daemon using:*
...

Reply:
done.